services/user.js

```diff
@@ -17,10 +17,34 @@ const isValidUserStatus = (userStatus) => {
 }
 
 class User {
-    constructor(userCollection, logCollection, organizationCollection) {
+    constructor(userCollection, logCollection, organizationService) {
```

Comment:
Same thing for this. Passing the `organizationService` was replaced in MVP-2 with `organizationCollection`. In MVP-2, we should be using `organizationCollection`.